dev-scheme/mit-scheme/mit-scheme-12.1-r1.ebuild

```diff
@@ -0,0 +1,94 @@
+# Copyright 1999-2024 Gentoo Authors
```

Comment:
You mean drop 11.2 or drop 12.1?

Reply:
> 12.1

the one you are revision-bumping, so 12.1 of course

Reply:
I can do that, but wouldn't that result in anyone with 12.1 currently installed (without `~amd64` set) being downgraded to 11.2 when they go to update their packages? Until the -r1 is stabled.

Reply:
> I can do that, but wouldn't that result in anyone with 12.1 currently installed (without `~amd64` set) being downgraded to 11.2 when they go to update their packages? Until the -r1 is stabled.

Well, yea, do not drop it to unstable. Sorry I did not saw it, you cna use satble kwds here.

Reply:
We do not need a stabilization-test window since this is a package fix, and I do not think there is any danger here to running into some unsupported case, is there?

Reply:
I'm not aware of any, just going off my understanding of the guidelines on [ebuild revisions](https://devmanual.gentoo.org/general-concepts/ebuild-revisions/index.html) and [keywording](https://devmanual.gentoo.org/keywording/index.html#keywording-on-upgrades).

Reply:
> I'm not aware of any, just going off my understanding of the guidelines on [ebuild revisions](https://devmanual.gentoo.org/general-concepts/ebuild-revisions/index.html) and [keywording](https://devmanual.gentoo.org/keywording/index.html#keywording-on-upgrades).

> This also applies to revision bumps, not just to upstream version changes.

Thats not really true. It is only done when the change is significient and mostly when it has not much to do with build fixes.

In this case the ONLY REAL reason this requires a revbump is because You are changing the dependencies.